tests/functional/simple_seed/test_seed.py

```diff
@@ -1,19 +1,290 @@
+import csv
+import os
 import pytest
-from dbt.tests.util import run_dbt
+
+from pathlib import Path
+
+from dbt.tests.util import (
+    run_dbt,
+    read_file,
+    check_relations_equal,
+    check_table_does_exist,
+    check_table_does_not_exist,
+)
+from tests.functional.simple_seed.fixtures import (
+    models__downstream_from_seed_actual,
+    models__from_basic_seed,
+    seeds__disabled_in_config,
+    seeds__enabled_in_config,
+    seeds__tricky,
+    seeds__wont_parse,
+)
 
 # from `test/integration/test_simple_seed`, test_postgres_simple_seed
 
 
-@pytest.fixture(scope="class")
-def project_config_update():
-    return {"seeds": {"quote_columns": False}}
+class SeedConfigBase(object):
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "config-version": 2,
+            "seeds": {
+                "quote_columns": False,
+            },
+        }
+
+
+class SeedTestBase(SeedConfigBase):
+    @pytest.fixture(scope="class", autouse=True)
+    def setUp(self, project):
+        """Create table for ensuring seeds and models used in tests build correctly"""
+        project.run_sql_file(project.test_data_dir / Path("seed_expected.sql"))
+
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        seed_actual_csv = read_file(dir_path, "seeds", "seed_actual.csv")
```

Comment:
Use project.test_data_dir, or test_data_dir, and put the seed in a 'data' directory.